src/View/Helper/FormHelper.php

```diff
@@ -1365,12 +1365,15 @@ protected function enumOptions(string $enumClass): array
     {
         assert(is_subclass_of($enumClass, BackedEnum::class));
 
+        $hasLabel = is_a($enumClass, EnumLabelInterface::class, true) || method_exists($enumClass, 'label');
```

Comment:
Do we really need both checks? We can just check for label method for both, right?

Reply:
The first is probably faster^^

Reply:
IIRC the double checks are cause the interface was added at bit later after the 5.0 release, I guess we can dropnthe method check in a new minor.

Reply:
I agree that one check would be faster than two, but I don't think we need to over-correct on efficiency here. Compatibility throughout the major is more important.